assets/js/displayMemberNames.js

```diff
@@ -0,0 +1,34 @@
+import $ from "jquery";
+
+function changeDisplayNames() {
+  const checkBox = document.getElementById("name_display_toggle");
+  const teamNamesCurrentlyDisplayed = !checkBox.checked;
```

Comment:
I'm not up to spec on best practices for JS, but this worked for me locally. Avoids the looping but no clue if its technically idiomatic.

```suggestion
  const teamNamesCurrentlyDisplayed = !document.getElementById("names").checked

  let styleSheet = document.getElementById("dynamicStyles");
  
  styleSheet.innerHTML = teamNamesCurrentlyDisplayed
      ? `.teamNames { display: block !important; } .memberNames { display: none !important; }`
      : `.teamNames { display: none !important; } .memberNames { display: block !important; }`;

```

Then you add this to pairing_display.html and pretty pairing.html
`<style id="dynamicStyles" type="text/css">
    </style>`